image/store_test.go

```diff
@@ -10,6 +10,14 @@ import (
 	"gotest.tools/v3/assert/cmp"
 )
 
+func TestCreate(t *testing.T){
```

Comment:
linter is failing on this line because it's not gofmt'ed

```
image/store_test.go:13: File is not `goimports`-ed (goimports)
```

```suggestion
func TestCreate(t *testing.T) {
```

Reply:
Thanks for pointing out this error 😅
I have fixed it!